tooling/v8-snapshot/package.json

```diff
@@ -40,19 +41,19 @@
   ],
   "types": "src/v8-snapshot.ts",
   "optionalDependencies": {
-    "@cypress/snapbuild-android-arm64": "1.0.3",
-    "@cypress/snapbuild-darwin-64": "1.0.3",
-    "@cypress/snapbuild-darwin-arm64": "1.0.3",
-    "@cypress/snapbuild-freebsd-64": "1.0.3",
-    "@cypress/snapbuild-freebsd-arm64": "1.0.3",
-    "@cypress/snapbuild-linux-32": "1.0.3",
-    "@cypress/snapbuild-linux-64": "1.0.3",
-    "@cypress/snapbuild-linux-arm": "1.0.3",
-    "@cypress/snapbuild-linux-arm64": "1.0.3",
-    "@cypress/snapbuild-linux-mips64le": "1.0.3",
-    "@cypress/snapbuild-linux-ppc64le": "1.0.3",
-    "@cypress/snapbuild-windows-32": "1.0.3",
-    "@cypress/snapbuild-windows-64": "1.0.3"
+    "@cypress/snapbuild-android-arm64": "1.0.4-beta-002",
```

Comment:
Will this stay beta?

Reply:
Sorry I should have clarified. I'll need to get the esbuild stuff in and fully released first (in non-beta form) and then I'll update this dependency with the fully released version. esbuild at:

https://github.com/cypress-io/esbuild/pull/14